src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/ConfirmServiceCallbackHandler.java

```diff
@@ -78,11 +80,8 @@
     private CallbackResponse prepopulateServedDocuments(CallbackParams callbackParams) {
         List<ServedDocuments> servedDocuments = List.of(ServedDocuments.CLAIM_FORM);
 
-        Map<String, Object> data = callbackParams.getRequest().getCaseDetails().getData();
-        data.put("servedDocuments", servedDocuments);
-
         return AboutToStartOrSubmitCallbackResponse.builder()
-            .data(data)
+            .data(Map.of("servedDocuments", servedDocuments))
```

Comment:
is the point of this PR not to remove bits like this?

Reply:
This is mid callback where we perpopulate the one field for the particular page. And this is the only use case for this kind of functionality. Sending the complete caseData doesn't seem to be right in this scenario.

Reply:
to be consistent across all changed that as well.